astropy/cosmology/_io/ecsv.py

```diff
@@ -190,9 +209,14 @@ def read_ecsv(
         the case of a merge conflict (e.g. for ``Cosmology(meta={'key':10}, key=42)``,
         the ``Cosmology.meta`` will be ``{'key': 10}``).
 
-    rename : dict or None (optional keyword-only)
-        A dictionary mapping column names to fields of the
-        `~astropy.cosmology.Cosmology`.
+    cosmology : str or type or None (optional, keyword-only)
+        The cosmology class (or string name thereof) to use when constructing the
+        cosmology instance. The class also provides default parameter values, filling in
+        any non-mandatory arguments missing in 'table'.
+
+    rename : dict or None (optional, keyword-only)
+        A dictionary mapping columns in 'table' to fields of the
```

Comment:
I find the original phrasing a bit clearer here
```suggestion
        A dictionary mapping column names in 'table' to fields of the
```